js/scripts/release.sh

```diff
@@ -89,11 +89,14 @@ fi
 
 echo "*** Updating cargo parity-ui-precompiled#$PRECOMPILED_HASH"
 git submodule update
+cat dapps/ui/Cargo.toml | sed "s/^parity-ui-precompiled = { git =.*/parity-ui-precompiled = { git = \"https:\/\/github.com\/ethcore\/js-precompiled.git\", optional = true, branch = \"$BRANCH\" }/" >dapps/ui/Cargo.toml.update
```

Comment:
Will update.

Actually reason for the split was that Mac sed is not 100% compatible, `-i` takes additional arguments. (And I wanted to test locally)